router/middleware/executors/executors.go

```diff
@@ -51,8 +53,26 @@ func Establish() gin.HandlerFunc {
 			return
 		}
 
-		// add the token to authenticate to the worker as a header
-		req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", c.MustGet("secret").(string)))
+		tm := c.MustGet("token-manager").(*token.Manager)
+
+		// set mint token options
+		mto := &token.MintTokenOpts{
+			Hostname:      "vela-server",
+			TokenType:     constants.WorkerAuthTokenType,
+			TokenDuration: time.Minute * 1,
```

Comment:
question: should we turn the vela-server token duration into a constant somewhere or since we create this type of token in a couple of places, create a method somewhere?

Reply:
we'll come back to this if we add more code like this 👍🏼